src/js/components/index.js

```diff
@@ -10,8 +10,8 @@ export {WeatherIcon} from './weather-icon';
 export {Page} from './page';
 export {GlobalHeader, GlobalNav} from './global-header';
 export {PageHeader, Title, PageTabs, TabLink} from './page-header';
-export {Table} from './table';
 export {LogConsole} from './log/index';
+export {Table} from './Table';
```

Comment:
I was concerned about this case change, but I just tested merging this onto a throw-away clone master and it worked correctly on OSX so it's OK in this case, git was smart enough to do a delete then save, instead of a NOP rename for once.